### PR TITLE
Improvment acceptance test issue 646

### PR DESCRIPTION
**Pull request by melowe**
Tests all run every time without flakes. Added checks that all servers are up and running before starting tests. Remove ServerSocket object as was overly complex and cumbersome.

The transport can be inferred by the scheme.

### CODE REVIEW

**Comment by melowe** [comment]
7 nodes samples and any other configurations will need to change to remove the server socket block.

**Review by chris-j-h — DISMISSED** [review]

**Comment by melowe** [comment]
Created PR on quorum examples to change `tessera-init.sh` to generate configs with new server descriptors.

https://github.com/jpmorganchase/quorum-examples/pull/167

File: tests/acceptance-test/src/test/java/config/ConfigGenerator.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        Path ouputFile = path.resolve(filename);
+        try (OutputStream out = Files.newOutputStream(ouputFile)){
+            JaxbUtil.marshalWithNoValidation(whiteListConfig, out);
+        } catch (IOException ex) {
+            throw new UncheckedIOException(ex);
+        }
+
+    }
+
+    public Path calclatePath(ExecutionContext executionContext) {
```
Comment (nicolae-leonte-go) [inline tests/acceptance-test/src/test/java/config/ConfigGenerator.java:62]: calculate? :)

Comment (melowe) [reply tests/acceptance-test/src/test/java/config/ConfigGenerator.java:62]: Yes

**Review by prd-fox — APPROVED** [review]

**Pull request merged**